pages/entity/_type/_.vue

```diff
@@ -140,7 +153,7 @@
 
       const contentfulClient = createClient(query.mode);
 
-      return axios.all([
+      const requests = [
```

Comment:
I think this assignment is causing all the requests to be made twice, you can observe this by going to: http://localhost:3000/fr/entity/topic/1669-color-negatives and with the network inspect open clicking the first entity pill that links to the photograph page.